python/ray/data/_internal/execution/operators/actor_pool_map_operator.py

```diff
@@ -760,20 +798,20 @@ def num_free_slots(self) -> int:
             for running_actor in self._running_actors.values()
         )
 
-    def _kill_inactive_actor(self) -> bool:
+    def _release_inactive_actor(self) -> bool:
```

Comment:
nit, I prefer using the word "remove". 
Because both `kill` and `release` are implementation details that can change in the future. 
also, please update the comments as well